### DISCUSSION TITLE
Questions about logo

### DISCUSSION THREAD

**Discussion by luc-tielen** [souffle-lang/souffle#2296] [General]
Hi!

I really like the souffle logo style, and was wondering who created it?
I would like a similar style logo for [Eclair Datalog](https://github.com/luc-tielen/eclair-lang), which I created.
The language is heavily inspired by souffle, so it would make sense if the logo is in the same style :smiley:.

**Comment by b-scholz** [comment]
Sorry for the delayed response. It has been a long time again since this logo was created. I cannot recall the details anymore (NB: somebody else created this logo).

**Reply by luc-tielen** [reply]
That's OK @b-scholz, it was worth a shot. In the meantime somebody volunteered to help create me a logo in a similar style (so stay tuned I guess :smiley:).
